### PR TITLE
Response body size calculation bug

### PR DESCRIPTION
**Pull request by tmkarthi**
Response body height was set to raw response height instead of beautified response height. The line height used was also wrong.

### CODE REVIEW

**Comment by sichvoge** [comment]
@cesaraugustogarcia obviously thats a very past PR but the problem with the wrong calculation seems to be still there. Unfortunately, there is no example / image attached so that we can compare. Can you have a quick look pls

File: src/app/directives/sidebar.js [hunk 1]
```diff
@@ -84,8 +84,8 @@
           var editorHeight = 50;
 
           if (jqXhr.responseText) {
```
Comment (llattes) [inline src/app/directives/sidebar.js]: Why is this check still here if then the `jqXhr.responseText` is not used anywhere?

File: src/app/directives/sidebar.js [hunk 2]
```diff
@@ -84,8 +84,8 @@
...
-            var lines = jqXhr.responseText.split('\n').length;
-            editorHeight = lines > 100 ? 2000 : 25*lines;
+            var lines = $scope.response.body.split('\n').length;
+            editorHeight = lines > 100 ? 2000 : Math.ceil(20.3 * lines);
```
Comment (llattes) [inline src/app/directives/sidebar.js]: This looks like an absolute magic number? What/where's the explanation for it?

**Comment by tmkarthi** [comment]
@llattes Files for reproducing the issue can be found in https://gist.github.com/tmkarthi/1a8e6e2aa392805a60e7685b10ce122c.

Steps to reproduce:
1. Place those 3 files in `dist/examples` directory in api-console repo.
2. Open api-console by running a file server at http://localhost:8080 with dist directory as the base directory.
3. Open both GET resources & click GET button in Try it.

Issues:
1. with_issue.json content is only partially visible. This is because the height of that display area is calculated based on the number of lines in the received text, while the beautified text has many more lines due to line wrapping.
2. proper.json content has a lot of empty space at the end. This is because 25 is used as the height of a line in calculation, whereas the actual height is 20. (Note: I no longer remember why I used 20.3. The line height in browser with the current CSS is 20. So, that should be fine.)

**Comment by tmkarthi** [comment]
Looks like the first issue (counting lines in raw response text vs. beautified text) was already fixed. The height calculation using 25 instead of 20 is still though.

**Comment by tmkarthi** [comment]
As only a very minor cosmetic issue is still open, closing this PR.

**Pull request closed without merging**